BackofficeBundle/Resources/config/field_type.yml

```diff
@@ -111,9 +111,9 @@ parameters:
                     required: true
         date:
             label: open_orchestra_backoffice.form.field_type.custom_type.date
-            type: date
+            type: date_picker
```

Comment:
accord with my previous remark, i think it's better to add a new field type for only datePicker